src/inc/eventtracebase.h

```diff
@@ -70,6 +70,69 @@ enum EtwThreadFlags
     kEtwThreadFlagThreadPoolWorker =  0x00000004,
 };
 
+#ifndef FEATURE_REDHAWK
+
+#if defined(FEATURE_EVENT_TRACE) 
+#if  !defined(FEATURE_PAL)
```

Comment:
Is the goal to only compile this for Windows or exclude it from the PAL build even on Unix platforms?